guides/source/action_mailer_basics.md

```diff
@@ -906,11 +907,12 @@ class EmailDeliveryObserver
 end
 ```
 
-Similar to interceptors, you must register observers using [`register_observer`][]. You can do this in an initializer file
-like `config/initializers/email_delivery_observer.rb`:
```

Comment:
@Bhacaz Does this mean we would no longer be able to use an initializer to register observers? I think this would break existing applications, no?

Reply:
@zzak 
`ActionMailer::Base.register_*` methods are at the same level as before and can still be use in an initializer. So it doesn’t introduce any breaking changes. 

The changes in documentation only intend to change the suggestion of the way or where registration can be done and give a more mailer callbacks of controller filters flavour to observers and interceptors.

Maybe the thing to do is presenting all possible ways to use this feature.

Reply:
@Bhacaz in that case I think we shouldn't remove it from the guide and keep your suggestion as preferred method going forward. Nice work btw!

Reply:
Sure 😄. Thank you for your feedback.